Build.mak

```diff
@@ -2,6 +2,10 @@ export ASSERT_ON_STOMPING_PREVENTION=1
 
 ifneq ($(DVER),1)
     DC := dmd-transitional
+
+	ifeq ($F, production)
+		override DFLAGS += -release
+	endif
```

Comment:
You're mixing tabs with spaces here.

Reply:
:scream:

Reply:
Really? Where?

Reply:
All other lines have spaces, only these few lines are tabs.

Reply:
Sorry, I looked closer, this is the line with spaces:

```
    DC := dmd-transitional
```

could you fix it in another commit?

Reply:
Updated.